errors/next-headers-wrong-context.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: next/headers methods were called outside request/render scope
+---
+
+#### Why This Error Occurred
+
+Either `headers()` or `cookies()` were invokned at place where no request information was available (eg.: Global scope, outside the React lifecycle).
```

Comment:
react lifecycle is not required, such as app route handler

```suggestion
Either `headers()` or `cookies()` were invoked somewhere outside of the request scope (e.g. Global scope).
```

Reply:
See https://github.com/vercel/next.js/pull/62143#discussion_r1493464969